File: airline_routes/v1/get/Java SDK/AirlineRoutes.java

// How to install the library at https://github.com/waconvo/waconvo-java
import com.waconvo.WAConvo;
import com.waconvo.exceptions.ResponseException;
import com.waconvo.resources.Destination;

// What are the destinations served by the British Airways (BA)?
public class AirlineRoutes {
  public static void main(String[] args) throws ResponseException {

    WAConvo waconvo = WAConvo
        .builder("YOUR_WACONVO_API_KEY", "YOUR_WACONVO_API_SECRET")
        .build();

    // Set query parameters
    Params params = Params
        .with("airlineCode", "BA");

    // Run the query
    Destination[] destinations = waconvo.airline.destinations.get(params);

    if (destinations[0].getResponse().getStatusCode() != 200) {
      System.out.println("Wrong status code: " + destinations[0].getResponse().getStatusCode());
      System.exit(-1);
    }

    System.out.println(destinations[0]);
  }
}

File: airline_routes/v1/get/Node SDK/airline_routes.js
const WAConvo = require("waconvo");

const waconvo = new WAConvo({
  clientId: "YOUR_WACONVO_API_KEY",
  clientSecret: "YOUR_WACONVO_API_SECRET",
});
// Or `const waconvo = new WAConvo()` if the environment variables are set

async function main() {
  try {
    // What are the destinations served by the British Airways (BA)?
    const response = await waconvo.airline.destinations.get({
      airlineCode: "BA",
    });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();
